app-crypt/trousers/files/trousers-0.3.15-tspi-drop-the-use-of-getpwent_r.patch

```diff
@@ -0,0 +1,67 @@
+From 2299eadf77ae7a7ec52148b6a8f4ea37e217eafa Mon Sep 17 00:00:00 2001
```

Comment:
Please send it upstream if you didn't already and include the link at the top of the patch.

Please also include a link to the relevant Gentoo bug at the top of the patch.

Reply:
Gentoo link added. Unfortunately my upstream submission is still awaiting moderator approval.